### PR TITLE
add containerd-netd testgrid

### PR DESCRIPTION
**Pull request by cadmuxe**
once this is working, I will add the testgrid for containerd&netd&calico.

### CODE REVIEW

File: config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml [hunk 1]
```diff
@@ -688,3 +709,32 @@ periodics:
...
         value: calico
+
+- interval: 2h
+  agent: kubernetes
+  name: ci-kubernetes-e2e-containerd-gce-netd
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-kubernetes-e2e-enable-netd: "true"
+    preset-kubernetes-e2e-containerd: "true"
```
Comment (cadmuxe) [inline config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: I added one more preset labels, will this work? @krzyzacy

Comment (krzyzacy) [reply config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: as long as it's not conflicting with other preset labels :-)

**Comment by krzyzacy** [comment]
^^ https://github.com/kubernetes/test-infra/issues/8886

/shrug

**Pull request merged**